src/constants.ts

```diff
@@ -174,11 +174,10 @@ export const MAC = 'MacOS'
 export const WINDOWS = 'Windows'
 export const LINUX = 'Linux'
 
-// for now, this will be constant values
-export const CURRENCIES = {
-  USD: 1583.42,
-  EUR: 1323.61,
-}
+export const INITIAL_CURRENCY = 'usd'
+export const LOADING = 'Loading...'
+export const COIN_GECKO_API_KEY = 'CG-FPQgFNTF26FzyCA7vtbaSh5U'
```

Comment:
Add key to the `.env` file and use it like that:
```suggestion
export const COIN_GECKO_API_KEY = process.env['COIN_GECKO_API_KEY']
```

Reply:
dj!